src/nvim/syntax_defs.h

```diff
@@ -73,4 +73,14 @@ typedef struct attr_entry {
   int cterm_fg_color, cterm_bg_color;
 } attrentry_T;
 
+#define ATTRENTRY_INIT {.rgb_ae_attr = 0, .cterm_ae_attr = 0, \
```

Comment:
Why not put `rgb_ae_attr` and `cterm_ae_attr` on their own lines too?  It looks a little odd having these two here but everything else on its own line.